CommunityFund/app/controllers/registrations_controller.rb

```diff
@@ -1,7 +1,60 @@
 class RegistrationsController < Devise::RegistrationsController
-  protected
+  before_action :configure_permitted_parameters
+
+  def edit
+    prep_geo_communities
+    super
+  end
 
-  def after_sign_up_path_for(resource)
-    '/dashboard/collect_information'
+  def update
+    prep_geo_communities
+    super
   end
-end
\ No newline at end of file
+
+  protected
+    def prep_geo_communities
+      if @countries.blank?
+        country_data = GeoInfo::countries.map { |c| [c.country, c.id] }
+        @countries = Hash[*country_data.flatten]
+      end
+
+      community = self.resource.geo_community
+      if not community.nil?
```

Comment:
Please use `blank?` for this if it's returning an active record. May be an empty array which is not nil

![screen shot 2015-03-11 at 11 43 05 pm](https://cloud.githubusercontent.com/assets/2533735/6625039/df8c5a26-c8c3-11e4-99d9-ce49165eb5f5.png)